BackEnd/src/main/java/com/exadel/carpoolfree/model/RatedPassenger.java

```diff
@@ -0,0 +1,39 @@
+package com.exadel.carpoolfree.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "passenger_rating")
+public class RatedPassenger {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "mark")
+    private short mark;
+
+    @ManyToOne
+    @JoinColumn(name = "drive_id")
+    private Drive drive;
+
+    @OneToOne
+    @JoinColumn(name = "passenger_id")
```

Comment:
Can we please create a test for this to check if `Driver` can put marks for some `Passengers` and those values are read correctly. And vise versa when `Passenger` rates `Driver`.